benchmarks/continuous.py

```diff
@@ -95,6 +96,31 @@ def bench(self, _code, _xdg_home):
     return [Result(self.name, 'runtime', avg_time)]
 
 
+@dataclass
+class PythonSubprocess:
+  name: str
+  repeats: int
+  variant: str = 'latest'
+  baseline_commit: str = BASELINE  # Unused by demanded by the driver
+
+  def clean(self, code, xdg_home):
+    run(code / 'dex', 'clean', env={'XDG_CACHE_HOME': Path(xdg_home) / self.variant})
+
+  def bench(self, code, xdg_home):
+    dex_py_path = code / 'python'
```

Comment:
How does that work? You never `build-ffi`, so the `libDex.so` will never be in that directory

Reply:
Good catch.  Implemented a more hermetic build, which coincidentally is also more locally testable because it no longer reads state from the checkout of the `continuous.py` script itself.